.github/workflows/lint.yml

```diff
@@ -15,17 +15,18 @@ jobs:
 
     steps:
       - name: Check out Git repository
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Set up Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v5
         with:
           python-version: 3.8
 
       - name: Install Python dependencies
-        run: pip install flake8
+        run: pip install flake8 black
 
       - name: Run linters
         uses: wearerequired/lint-action@v2
         with:
+          black: true
           flake8: true
```

Comment:
Should we only call black then, and not flake8?

Reply:
This is just for checking, not fixing. So I don't mind running both. For fixing syntax we should make a choice.